pkg/ui/handlers.go

```diff
@@ -1,23 +1,32 @@
 package ui
 
 import (
+	"fmt"
 	"io/fs"
 	"net/http"
 	"path"
+	"regexp"
+	"strings"
 
 	"github.com/canonical/identity-platform-login-ui/internal/logging"
 	"github.com/go-chi/chi/v5"
 )
 
 type API struct {
 	fileServer http.Handler
-
-	logger logging.LoggerInterface
+	fileSystem fs.FS
+	logger     logging.LoggerInterface
 }
 
 func (a *API) RegisterEndpoints(mux *chi.Mux) {
 	// TODO @shipperizer unsure if we deal with any POST/PUT/PATCH via js
-	mux.HandleFunc("/*", a.uiFiles)
+	nodeList, _ := fs.ReadDir(a.fileSystem, ".")
+	r, _ := regexp.Compile(`html$`)
```

Comment:
This does not work, we need to serve other types of files as well like `.js`, `.css` and images. If you go to the `login` page you will see that it does not load.

Reply:
fixed